controller/controller.go

```diff
@@ -1235,6 +1235,7 @@ func (ctrl *ProvisionController) shouldProvision(ctx context.Context, claim *v1.
 // shouldDelete returns whether a volume should have its backing volume
 // deleted, i.e. whether a Delete is "desired"
 func (ctrl *ProvisionController) shouldDelete(ctx context.Context, volume *v1.PersistentVolume) bool {
```

Comment:
Do you want to log the return value?

Reply:
Added return value to each log statement.